examples/hubtype_demo/src/actions/wait.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+
+export default class extends React.Component {
+  render() {
+    return <></>
+  }
+}
```

Comment:
what is doing this action, can be removed?

Reply:
It's a requirement that Marc wanted, after a handoff, de bot don't reply, just ignore the return, I remove the file, and put this behaviour inside `routes.js`